_posts/2025-03-05-next-scala-lts-jdk.md.md

```diff
@@ -0,0 +1,17 @@
+---
+category: announcement
+permalink: /news/next-scala-lts-jdk.html
+title: "JDK 17 will be the next JDK version used by Scala 3"
```

Comment:
```suggestion
title: "JDK 17 will be the next minimum version required by Scala 3"
```

Reply:
another possibility would be to flip it around: "Scala 3 will begin requiring JDK 17+" or "Future Scala 3 versions will require JDK 17+"